mrack.spec

```diff
@@ -0,0 +1,61 @@
+# Created by pyp2rpm-3.3.5
+%global srcname mrack
+
+Name:           %{srcname}
+Version:        0.6.0
```

Comment:
We need a way to have this automatically bumped in a release process.

Reply:
This can be achieved by adding this to `[tool.semantic_release]` section of the `pyproject.toml` file:
```ini
version_pattern = [
    "mrack.spec:Version:\\s+{version}",
]
```
But I'm still trying to automate the changelog portion.